modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -510,3 +510,9 @@
 	slot = SLOT_IN_BACKPACK
 	path = /obj/item/bedsheet/cosmos
 	ckeywhitelist = list("grunnyyy")
+
+/datum/gear/donator/customskirt
+	name = "custom atmos skirt"
+	slot = SLOT_IN_BACKPACK
```

Comment:
pretty sure this should be SLOT_W_UNIFORM because if it can't go on the person it automatically gets placed in the backpack slot

Reply:
Keeping it in backpack.